app/templates/views/guidance/using-notify/attach-pages.html

```diff
@@ -0,0 +1,45 @@
+{% extends "content_template.html" %}
+
+{% from "components/service-link.html" import service_link %}
+
+{# Used by the content_template.html layout, prefixes the "navigation" accessible name #}
+{% set navigation_label_prefix = 'Using Notify' %}
+
+{% block per_page_title %}
+  Attach pages
```

Comment:
```suggestion
  Attach pages to a letter template
```

The H1 could probably work harder to describe what we‘re attaching pages to. Outside the context of the Notify UI we need to make sure this stuff is distinct from uploading a letter.

I think the navigation can stay as `Attach pages` - especially if we’re going to start grouping the links on the Using Notify page.